certbot/tests/client_test.py

```diff
@@ -18,12 +18,13 @@
 KEY = test_util.load_vector("rsa512_key.pem")
 CSR_SAN = test_util.load_vector("csr-san_512.pem")
 
+# pylint: disable=line-too-long
```

Comment:
Gah I wish we didn't have to do this but I think this is better than cleaning up all of these lines right now.